src/app/search/search.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="main">
+  <section>
+    <header>
+      <h2>Do a search for...</h2>
+
+      <div class="form-group">
+        <input (keyup)="search()" [(ngModel)]="searchText" placeholder="search by title or description"/><!-- TODO: delay search until finish typing -->
```

Comment:
In fact, I sometimes get search results back out-of-order when typing.  May even need to cancel any in-flight search requests when you change your search here.  Though the CMS stuff doesn't support that, currently.